src/reconcile/rec_write.c

```diff
@@ -2459,15 +2459,15 @@ __rec_write_wrapup(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page)
         if (__wt_ref_is_root(ref))
             break;
 
-        WT_RET(__wt_ref_block_free(session, ref));
+        ret = __wt_ref_block_free(session, ref);
```

Comment:
Why we need to do this?

Reply:
debug changes - removed here [1c4f7bb](https://github.com/wiredtiger/wiredtiger/pull/10686/commits/1c4f7bb287446e1166c6d19da2501a9c365bff50)